planet-express/java/src/Crew.java

```diff
@@ -0,0 +1,74 @@
+
+public class Crew {
+	int thirst = 50;
```

Comment:
I'm not sure I understand what you mean by using constants. Isn't the game supposed to change the value of the thirst, horde, etc? How can those values change if they're constants?